Pagination/PaginatedList.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\RadBundle\Pagination;
+
+
+/**
+ * Generic value object for returning paginated lists
+ */
+class PaginatedList
+{
+    /**
+     * @var array
+     */
+    private $list;
+
+
+    /**
+     * @var Pagination
+     */
+    private $pagination;
+
+
+    public function __construct (array $list, Pagination $pagination)
```

Comment:
Do we want to use `iterable` instead of `array`?

```suggestion
    public function __construct (iterable $list, Pagination $pagination)
```

Reply:
Good idea.